seth-sethgrosenberg/hw_01/4-homework-99-bottles-of-beer/app.js

```diff
@@ -0,0 +1,26 @@
+/*
+  Write a script that prints the lyrics to "99 Bottles of Beer on the Wall" in the terminal. 
+  If you're unfamiliar with the song, you can [find the lyrics here](http://www.99-bottles-of-beer.net/lyrics.html).
+  
+  Be sure that all of your output includes proper spacing (no words or values running into each other).
+
+  Hint: You can make your output appear on multiple lines by using the string \n to insert a line break.
+  
+  For an extra challenge, improve this code to print "1 bottle" rather than "1 bottles." (Hint: read up on if/else
+  conditional statements at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else). 
+
+*/
+
+for (var i=99, j=98; i >= 1 && j >= 1; i--, j--) {
+	if (i>1,j>1) {
+		console.log(i + " bottles of beer on the wall, " + i + " bottles of beer." + "\n"); 
+		console.log("Take one down and pass it around, " + j + " bottles of beer on the wall." + "\n");
```

Comment:
Is using two iterators easier conceptually for you to keep track of the value of `i`? I encourage you to use just one iterator, and manipulate it with string interpolation as needed. Example for line 17:
```console.log("Take one down and pass it around, " + (i -1) + " bottles of beer on the wall." + "\n");```

Neat trick, though. :P